uninstall.ps1

```diff
@@ -4,26 +4,29 @@ $env:PYTHONIOENCODING = "UTF-8"
 if($env:VIRTUAL_ENV){
     $NNI_PYTHON3 = $env:VIRTUAL_ENV + "\Scripts"
     $NNI_PKG_FOLDER = $env:VIRTUAL_ENV + "\nni"
-    Remove-Item "$NNI_PYTHON3\node.exe" -Force
+    cmd /c del "$NNI_PYTHON3\node.exe"
 }
 else{
     $NNI_PYTHON3 = $(python -c 'import site; from pathlib import Path; print(Path(site.getsitepackages()[0]))')
     $NNI_PKG_FOLDER = $NNI_PYTHON3 + "\nni"
-    Remove-Item "$NNI_PYTHON3\Scripts\node.exe" -Force
+    cmd /c del "$NNI_PYTHON3\Scripts\node.exe"
 }
 
 $PIP_UNINSTALL = """$NNI_PYTHON3\python"" -m pip uninstall -y "
 $NNI_NODE_FOLDER = $NNI_DEPENDENCY_FOLDER+"\nni-node"
 $NNI_YARN_FOLDER = $NNI_DEPENDENCY_FOLDER+"\nni-yarn"
 
 # uninstall
-Remove-Item $NNI_PKG_FOLDER -Recurse -Force
-cmd /C $PIP_UNINSTALL "nni"
+cmd /c rmdir /s /q $NNI_PKG_FOLDER
+cmd /c $PIP_UNINSTALL "nni"
 
-# clean
-Remove-Item "src/nni_manager/dist" -Recurse -Force
-Remove-Item "src/nni_manager/node_modules" -Recurse -Force
-Remove-Item "src/webui/build" -Recurse -Force
-Remove-Item "src/webui/node_modules" -Recurse -Force
-Remove-Item $NNI_YARN_FOLDER -Recurse -Force
-Remove-Item $NNI_NODE_FOLDER -Recurse -Force
+# clean up
+cmd /c rmdir /s /q "build"
```

Comment:
What's the drawback of `Remove-Item`?

Reply:
Remove-item doesn't work for symbol links. If use powershell to remove a symbol link, it needs to call like (Get-Item xxx).Delete(). But the Delete() doesn't work for non-empty physical folder.